src/vscode.d.ts

```diff
@@ -0,0 +1,9 @@
+import { WorkspaceConfiguration } from 'vscode';
+import { IVSIcons } from '../contributions';
+
+declare module 'vscode' {
+  interface WorkspaceConfiguration {
+    vsicons: IVSIcons;
+    inspect<T>(section: string): {defaultValue: T, globalValue: T, key: string, workspaceValue: T} | undefined;
+  }
+}
```

Comment:
1. Workspace always wins except in file and folder associations, where a merge is done and removed identical entries.

2. As we're not using any of this in the tests I thought to provide a more natural way of dealing with de vscode api. I would consider having the `IExtendedWorkspaceConfiguration` if needed in the tests, which it seems not to be the case for the moment.